src/Loggers/Loggers.cpp

```diff
@@ -373,6 +411,14 @@ void Loggers::updateLevels(Poco::Util::AbstractConfiguration & config, Poco::Log
                     const std::string name(config.getString("logger.levels." + key + ".name"));
                     const std::string level(config.getString("logger.levels." + key + ".level"));
                     logger.root().get(name).setLevel(level);
+
+                    std::string pos_pattern = config.getRawString("logger.levels." + key + "message_regexp", global_pos_pattern);
+                    std::string neg_pattern = config.getRawString("logger.levels." + key + "message_regexp_negative", global_neg_pattern);
```

Comment:
What is expected behavior when I specify a global positive pattern and a level specific negativ pattern?

Reply:
Both the global positive pattern and level-specific negative pattern should apply for the expression